### DISCUSSION TITLE
connect project host like jetbrains

### DISCUSSION THREAD

**Discussion by ahmedbondok2019** [microsoft/vscode-discussions#818] [Extension Development QnA]
i can connect to to project host one connection for one project
i can upload files with out any another app like filezilla
just right click and deploy this file using map for project to upload to the same place
it will be good addition

**Comment by olguzzar** [comment]
Hi @ahmedbondok2019, this forum is specifically for extension development. For any VS Code feature requests, please submit a new issue on the [main VS Code repo](https://github.com/microsoft/vscode/issues). Thanks for your post, and I'm now going to close this discussion as off-topic!
